website_cookie_notice/models/res_company.py

```diff
@@ -19,57 +19,61 @@
 #
 ##############################################################################
 
-from openerp import models, fields
-from openerp.tools.translate import _
+from openerp import _, models, fields
 
 
 class Company(models.Model):
     _inherit = "res.company"
     cookieAnalytics = fields.Boolean(
```

Comment:
field names should be snake_case (this is not your fault), but please change them to this convention and use oldname to have automatically migrated the columns (bump also the version y number, please)